.circleci/config.yml

```diff
@@ -0,0 +1,155 @@
+version: 2.1
+
+###
+# Parameters
+###
+parameters:
+  pr_workflow:
+    type: boolean
+    default: true # by default pr workflow will get executed.
+
+###
+# Executors
+###
+executors:
+  default:
+    docker:
+      - image: cimg/node:lts
```

Comment:
Do we really want to pin on lts? This will break future builds if new lts versions are not compatible with our app. I think we want this to be manual..